docs/layers.md

```diff
@@ -867,7 +867,9 @@ The values of population rank are derived from the `population` value as follows
 * 3: 1k to 2k
 * 2: 200 to 1k
 * 1: Less than 200
-* 0: No `population` value available.
+* 0: No `population` value available or `population` value zero.
+
+OSM localities gets their population_rank from NaturalEarth's `pop_max` tag because NaturalEarth `pop_max` is for the metro area and is more useful for label grading in the stylesheet.
```

Comment:
When available, for the largest cities OSM localities...`

Reply:
See also:

- https://github.com/whosonfirst/whosonfirst-properties/tree/main/properties/wof#population_rank